index.js

```diff
@@ -1,7 +1,18 @@
-const emailLog = require('./email_log.json');
+const emailLog = require("./email_log.json");
 
 function countUniqueEmails(emailFile) {
   // do work here
+
+  let newObj = emailFile.emails.reduce(function(p, c) {
```

Comment:
You can return the `reduce` function directly instead of assigning it to a new variable and returning that. Example:
`return emailFIle.emails.reduce(...)`